inst/include/population_dynamics/fleet/fleet.hpp

```diff
@@ -153,7 +153,7 @@ struct Fleet : public FIMSObject<Type> {
     size_t dims = this->observed_agecomp_data->get_imax() *
                   this->observed_agecomp_data->get_jmax();
     if (dims != this->catch_numbers_at_age.size()) {
-      fims_log::get("fleet.log") << "Error: observed age comp is of size "
+      FLEET_LOG << "Error: observed age comp is of size "
```

Comment:
Probably, I had only added those in the create model loop but seems valid here. I wonder if this error is actually able to be thrown or if it will be caught first in create model?

Reply:
As per the comment from @msupernaw on issue #506 we may not want these exit calls. Are we able to update the valid_model parameter in information from here? information->valid_model=FALSE; or something like that?? I don't know if we can pass values in that direction?

Reply:
Reviewing on the seaside chat - it doesn't look like we have access to information from fleet. Peter, Nathan, and I thought since the `exit(1)` calls are already in there and this issue has been open for a while, we would try it out with the exit functionality and then go back to only using `valid_model` if it becomes too annoying that the program quits mid-run when debugging in M2.